README.md

```diff
@@ -11,20 +11,20 @@
 
 # What is Rook?
 
-Rook is an open source **cloud-native storage orchestrator** for Kubernetes, providing the platform, framework, and support for a diverse set of storage solutions to natively integrate with cloud-native environments.
+Rook is an open source **cloud-native storage orchestrator** for Kubernetes, providing the platform, framework, and support for Ceph storage to natively integrate with Kubernetes.
 
-Rook turns storage software into self-managing, self-scaling, and self-healing storage services. It does this by automating deployment, bootstrapping, configuration, provisioning, scaling, upgrading, migration, disaster recovery, monitoring, and resource management. Rook uses the facilities provided by the underlying cloud-native container management, scheduling and orchestration platform to perform its duties.
```

Comment:
I think we should keep this 16 and 18 line,
This makes me more understandable what Rook can do,
Btw I used this line on my college internship project report XD

Reply:
ohh In see 16 the is preserved,

But 18 can be also there with little modifications
```Rook provides a seamless experience for scheduling, lifecycle management, resource management, security, monitoring, and user experience.```

Reply:
There are two sentences (lines 18-19) that describe how rook manages Ceph. I guess the previous line 18 seems redundant to me after those other two sentences.